modules/apps/portal-cache/portal-cache-impl/src/main/java/com/liferay/portal/cache/internal/dao/orm/EntityCacheImpl.java

```diff
@@ -260,10 +258,27 @@ public void removeResult(Class<?> clazz, Serializable primaryKey) {
 		_removeResult(clazz, primaryKey, null);
 	}
 
+	public class EntityCacheCacheRegistryItem implements CacheRegistryItem {
+
+		@Override
+		public String getRegistryName() {
+			return EntityCache.class.getName();
+		}
+
+		@Override
+		public void invalidate() {
+		}
```

Comment:
Hey @GabrielHenriquedev, you must call the `clearCache` method here as well. Thx